docs/readmes/contributing/contribute_conventions.md

```diff
@@ -194,6 +194,53 @@ Orc8r's cloud code has some basic [CI lint checks](https://github.com/magma/magm
 - When import aliasing is required, prefer to alias with `snake_case` rather than `camelCase`
 - Prefer readable code over rigid adherence to max line lengths. Capping around 140 characters feels about right.
 
+### Python
+
+In general, we follow the [PEP 8 style guide](https://www.python.org/dev/peps/pep-0008/). 
+
+**Type Annotation**
+
+- All new code should be fully type-annotated. 
```

Comment:
General nit: don't terminate your list items with periods unless you have more than one sentence in the item